regcomp.c

```diff
@@ -10752,7 +10752,8 @@ S_regclass(pTHX_ RExC_state_t *pRExC_state, I32 *flagp, U32 depth,
                     }
                     first_time = FALSE;
 
-                    sv_catpv(substitute_parse, SvPVX(this_sequence));
+                    sv_catpvn(substitute_parse,
```

Comment:
Isn't this done better with `sv_catsv`?

Reply:
@leont I am not so sure. This code is effectively using AV's and SV's for their "self cleanup" properties. As far as I could see these are "normal" SVPV's constructed by the regex compiler, and as such use sv_catpv() would just do a bunch of checks we know will be false. This is one of the few cases where using SvPVX() makes sense to me.